src/lib/components/Navbar.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+  import GithubStats from './GithubStats.svelte';
+  import Pill from '$lib/design-components/Pill.svelte';
+  import tempLogo from '$lib/images/temp-logo.svg';
+  import { theme } from '$lib/theme';
+  import backButton from '$lib/images/back-button.svg';
+  import Flex from '$lib/design-components/Flex.svelte';
+</script>
+
+<div class="header-container">
+  <header>
+    <div class="corner">
+      test123
+      <img src={tempLogo} alt="GitHub" />
+    </div>
+
+    <nav>
+      <ul>
+        <li aria-current={$page.url.pathname === '/' ? 'page' : undefined}>
+          <a href="/">Home</a>
+        </li>
+        <li aria-current={$page.url.pathname === '/testpage2' ? 'page' : undefined}>
+          <a href="/testpage2">testpage2</a>
+        </li>
+        <li aria-current={$page.url.pathname.startsWith('/about') ? 'page' : undefined}>
+          <a href="/about">about</a>
+        </li>
+      </ul>
+    </nav>
+    <Flex direction="row" gap={2}>
+      <GithubStats />
+      <Pill color={theme.colors.orange500}>
+        <Flex direction="row" alignItems="center" gap={1}>
+          Get a demo
+          <img src={backButton} alt="Back" />
+        </Flex>
+      </Pill>
+    </Flex>
+  </header>
+</div>
+
+<style>
+  .header-container {
+    background-color: var(--color-light100);
+    max-width: 64rem;
+    margin: 0 auto;
+    padding: 1rem;
+    width: 100%;
+  }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  .corner {
+    height: 3em;
+    display: flex;
+    gap: 0.5rem;
```

Comment:
is there a specific reason you choose to us `em` and then `rem` does this make sense?